### PR TITLE
Add Sema's Discord in the contact_fields field

### LINKED ISSUE #3: [FEAT] - Add Sema's Discord in the `contact_fields` field
### What feature would you like to see?

**Description**

As added in other repositories, the Discord link should be added
in the `contact_fields` field in the issue template configuration.

_If you have already forked the repository, skip to step 3_

1. Fork the repository [here](https://github.com/Semalab/oswc-query-bot/fork)
2. Go to your fork recently created.
3. Make a new branch named `discord_link`.
4. Once you've created the branch, go to `.github/ISSUE_TEMPLATE`, and open `config.yml`.
5. At the bottom, add the following code below:

```yaml
contact_links:
  - name: Questions
    url: https://discord.gg/Byjr6rdBUZ
    about: Any questions or comments you have, let us know on our Discord server.
```

6. Make a PR in this repository. If not sure how, see the [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
7. You're done now! You just need to wait for the maintainers to review your PR and merge it.

If you need any help, comment below and we will help you. 🙂 

### Additional information

This is a good first issue. Please leave it open to beginners and first-time contributors. 🙂

### PR DESCRIPTION
**Pull request by aroramrinaal**
# Pull Request

## Description: 
Added the Discord link in the contact_fields field in the issue template configuration.

Closes #3

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
Thanks for taking care of this!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1 +1,5 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Questions
+    url: https://discord.gg/Byjr6rdBUZ
+    about: Any questions or comments you have, let us know on our Discord server.
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/config.yml:5]: There's a missing endline here. I believe this should be fixed.
```suggestion
    about: Any questions or comments you have, let us know on our Discord server.

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Comment (aroramrinaal) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Ya,I will add the missing **endline**.

Comment (aroramrinaal) [reply .github/ISSUE_TEMPLATE/config.yml:5]: > There's a missing endline here. I believe this should be fixed.
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix  |  **Tags:** Inefficient

Fixed it #6

**Pull request merged**